src/systemtask/SystemTask.cpp

```diff
@@ -425,6 +425,15 @@ void SystemTask::Work() {
         case Messages::BatteryPercentageUpdated:
           nimbleController.NotifyBatteryLevel(batteryController.PercentRemaining());
           break;
+        case Messages::LowBattery: {
+          Pinetime::Controllers::NotificationManager::Notification notif;
+          std::array<char, 101> message {"Low Battery\0Charge your watch to prevent data loss.\0"};
```

Comment:
Let's change <char, 101> to <char, Pinetime::Controllers::NotificationManager::MessageSize + 1>